backend/src/models/user.ts

```diff
@@ -0,0 +1,10 @@
+export type User = {
+  user_id: number;
+  email: string;
+  first_name: string;
+  passwd: string;
+  unallocated_funds: string;
+};
+
+export type CreateUserPayload = Omit<User, 'user_id'>;
+export type BaseUser = Omit<User, 'user_id'>;
```

Comment:
Base entity is the entity without the primary id field since the db will automatically populate it